sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/SASTest.groovy

```diff
@@ -1,1263 +0,0 @@
-import com.azure.storage.blob.models.UserDelegationKey
-import com.azure.storage.blob.sas.BlobContainerSasPermission
-import com.azure.storage.blob.sas.BlobSasPermission
-import com.azure.storage.blob.sas.BlobServiceSasSignatureValues
-import com.azure.storage.blob.sas.BlobSasServiceVersion
-import com.azure.storage.blob.specialized.SpecializedBlobClientBuilder
-import com.azure.storage.common.Utility
-import com.azure.storage.common.implementation.StorageImplUtils
-import com.azure.storage.common.sas.AccountSasPermission
-import com.azure.storage.common.sas.AccountSasResourceType
-import com.azure.storage.common.sas.AccountSasService
-import com.azure.storage.common.sas.AccountSasSignatureValues
-import com.azure.storage.common.sas.SasProtocol
-import com.azure.storage.common.StorageSharedKeyCredential
-
-import com.azure.storage.common.implementation.Constants
-import com.azure.storage.common.sas.SasIpRange
-import spock.lang.Ignore
-import spock.lang.Unroll
-
-import java.time.LocalDateTime
-import java.time.OffsetDateTime
-import java.time.ZoneOffset
-
-class SASTest extends APISpec {
-
-    // TODO (gapra) : Cleanup SAS Tests
-
-    @Unroll
-    def "Blob range"() {
-        expect:
-        if (count == null) {
-            assert new BlobRange(offset).toHeaderValue() == result
-        } else {
-            assert new BlobRange(offset, count).toHeaderValue() == result
-        }
-
-
-        where:
-        offset | count || result
-        0      | null  || null
-        0      | 5     || "bytes=0-4"
-        5      | 10    || "bytes=5-14"
-    }
-
-    @Unroll
-    def "Blob range IA"() {
-        when:
-        new BlobRange(offset, count)
-
-        then:
-        thrown(IllegalArgumentException)
-
-        where:
-        offset | count
-        -1     | 5
-        0      | -1
-    }
-
-    def "BlobClient getSnapshot"() {
-        setup:
-        def data = "test".getBytes()
-        def blobName = generateBlobName()
-        def bu = cc.getBlobClient(blobName).getBlockBlobClient()
-        bu.upload(new ByteArrayInputStream(data), data.length)
-        def snapshotId = bu.createSnapshot().getSnapshotId()
-
-        when:
-        def snapshotBlob = cc.getBlobClient(blobName, snapshotId).getBlockBlobClient()
-
-        then:
-        snapshotBlob.getSnapshotId() == snapshotId
-        bu.getSnapshotId() == null
-    }
-
-    def "BlobClient isSnapshot"() {
-        setup:
-        def data = "test".getBytes()
-        def blobName = generateBlobName()
-        def bu = cc.getBlobClient(blobName).getBlockBlobClient()
-        bu.upload(new ByteArrayInputStream(data), data.length)
-        def snapshotId = bu.createSnapshot().getSnapshotId()
-
-        when:
-        def snapshotBlob = cc.getBlobClient(blobName, snapshotId).getBlockBlobClient()
-
-        then:
-        snapshotBlob.isSnapshot()
-        !bu.isSnapshot()
-
-    }
-
-    def "serviceSASSignatureValues network test blob"() {
```

Comment:
Network tests deleted due to duplicate in SasClientTests that use the non-deprecated methods.